tsuru/client/plugin_test.go

```diff
@@ -24,6 +25,57 @@ func (s *S) TestPluginInstallInfo(c *check.C) {
 	c.Assert(PluginInstall{}.Info(), check.NotNil)
 }
 
+func (s *S) TestPluginInstallWithManifest(c *check.C) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "fakeplugin")
+	}))
+	defer ts.Close()
+	ts2 := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		urls := make(map[string]string)
+		urls["darwin/amd64"] = ts.URL
+		urls["darwin/arm64"] = ts.URL
+		urls["linux/amd64"] = ts.URL
+		urls["linux/arm64"] = ts.URL
+		urls["linux/386"] = ts.URL
+		urls["windows/amd64"] = ts.URL
+		urls["windows/386"] = ts.URL
+		manifest := PluginManifest{SchemaVersion: "1.0", Metadata: PluginManifestMetadata{Name: "rpaasv2", Version: "0.33.1"}, URLPerPlatform: urls}
```

Comment:
`Name: "myplugin",` for consistency with context below

Reply:
Fixed 😅